vpr/CMakeLists.txt

```diff
@@ -33,8 +33,8 @@ include(GlibCompileResourcesSupport)
 
 #Collect the source files
 file(GLOB_RECURSE EXEC_SOURCES src/main.cpp)
-file(GLOB_RECURSE LIB_SOURCES src/*/*.cpp)
-file(GLOB_RECURSE LIB_HEADERS src/*/*.h)
+file(GLOB_RECURSE LIB_SOURCES src/*/*.cpp src/*/*/*.cpp)
+file(GLOB_RECURSE LIB_HEADERS src/*/*.h src/*/*/*.h)
```

Comment:
I'd be hesitant to add too many levels of hierarchy to the source code.

If you want to put the new RR graph in a `device` directory I'd just make a single flat directory: `vpr/src/device` (no rr_graph subdirectory).

Reply:
Moved the rr_graph* to device directory. Reverted to the old CMakeList.